L/libwebsockets/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder, Pkg
+
+name = "libwebsockets"
+version = v"4.3.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/warmcat/libwebsockets.git", "5102a5c8d6110b25a01492fcf96fb668b13dd6e7"),
```

Comment:
That commit isn't the commit corresponding to v4.3.3: https://github.com/warmcat/libwebsockets/commit/4415e84c095857629863804e941b9e1c2e9347ef

Reply:
Oh, I accidentally grabbed the latest hash from the 4.3-stable branch 😅 
Fixed it now by adding the 4.3.3 tag hash!